Sources/ClientRuntime/Networking/Http/Middlewares/FlexibleChecksumsResponseMiddleware.swift

```diff
@@ -102,7 +111,7 @@ public struct FlexibleChecksumsResponseMiddleware<OperationStackOutput>: Middlew
             throw ClientError.dataNotFound("Cannot calculate the checksum of an empty body!")
         }
 
-        return try await next.handle(context: context, input: input)
+        return output
```

Comment:
Rearranged where `next.handle()` is called in this middleware so that multiple GET requests are not made to the server.